### PR TITLE
Julia Kozeeva Tasks 21-30

### PR DESCRIPTION
**Pull request by usersadvocate**

### CODE REVIEW

File: tasks/22_remove-falsy-values/remove-falsy-values.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function removeFalsyValues(arr) {
+  arr = arr.filter(function(element) {
```
Comment (alexkond) [inline tasks/22_remove-falsy-values/remove-falsy-values.js]: You're overwriting function argument, is it required by the task?

File: tasks/27_basic-subclasses/basic-subclasses.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class Human {
+  constructor(name, sex) {
+    this.name = name;
+    this.sex = sex;
+  }
+}
+class Man extends Human {
+  constructor(name, sex) {
```
Comment (alexkond) [inline tasks/27_basic-subclasses/basic-subclasses.js]: unused `sex` argument in the constructor

File: tasks/27_basic-subclasses/basic-subclasses.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    super(name, 'male');
+  }
+}
+class Woman extends Human {
+  constructor(name, sex) {
```
Comment (alexkond) [inline tasks/27_basic-subclasses/basic-subclasses.js]: unused `sex` argument in the constructor

**Review by alexkond — CHANGES_REQUESTED** [review]

File: tasks/22_remove-falsy-values/remove-falsy-values.js (hunk 1 above)
Comment (usersadvocate) [reply tasks/22_remove-falsy-values/remove-falsy-values.js]: no, this is my mistake I will fix it.

**Review by alexkond — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

File: tasks/22_remove-falsy-values/remove-falsy-values.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+function removeFalsyValues(arr) {
+  return arr.filter(function(element) {
+    return element;
```
Comment (KalachevDev) [inline tasks/22_remove-falsy-values/remove-falsy-values.js:3]: Could be simplified with Boolean

**Review by KalachevDev — APPROVED** [review]

**Pull request merged**